src/disrupt.sh

```diff
@@ -24,14 +24,21 @@ if [[ ! -d "${IMG_ASSET_DIR}" || ! $(find "${IMG_ASSET_DIR}" -mindepth 1 -iname
 fi
 
 #Check for an image associated with the fact
-ASSOCIATED_IMAGE=$("${RUN_DIR}/src/lf_imap.py ${RANDOM_FACT}}")
-if [ "$ASSOCIATED_IMAGE" ] && [ -s "$ASSOCIATED_IMAGE" ]; then
-	echo "Found a necessary image boss"
-elif [ "${DICE_ROLL}" -eq 1 ]; then
-	# add the image if no matching but random chance said to
-   	RANDOM_IMAGE=$(find "${IMG_ASSET_DIR}" -type f | shuf -n 1)
-   	cat "${MSG_TEMPLATE_IMG}" >> "${TEMP_FILE}"
-   	base64 "${RANDOM_IMAGE}" >> "${TEMP_FILE}"
+fact_cmd="$RUN_DIR/src/lf_imap.py $RANDOM_FACT"
+ASSOCIATED_IMAGE=$($fact_cmd)
+DICE_ROLL=${DICE_ROLL/.*}
+
+if [ "${DICE_ROLL}" -eq 1 ]; then
+	if [ "$ASSOCIATED_IMAGE" ] && [ -s "$ASSOCIATED_IMAGE" ]; then
+		echo "Using associated image"
+		RANDOM_IMAGE=$ASSOCIATED_IMAGE
+	else
+		# add the image if no matching but random chance said to
+		echo "Using random image"
+		RANDOM_IMAGE=$(find "${IMG_ASSET_DIR}" -type f | gshuf -n 1)
```

Comment:
I don't think this will work on the actual server, since it looks like you're using the GNU coreutils package that ``brew`` installs (unless you alias ``shuf`` to ``gshuf`` or symlink ``shuf`` to ``gshuf``).

```bash
brew install coreutils
```

installs the GNU coreutils with the ``g`` prefix, allowing you to use the Mac OS X utilities alongside with the GNU core utilities. Adding ``--default-names`` (or ``--with-default-names``) if your version of ``brew`` is new enough installs them without the prefix.

Reply:
accidentally committed here, interesting they have the default-names flag